pool/src/lib.rs

```diff
@@ -553,17 +596,25 @@ impl LendingPool {
             return Ok(false);
         }
 
-        let collat_coeff = Self::get_collateral_coeff(env, reserve)?;
+        let collat_coeff = Self::get_collat_coeff(env, &asset, reserve)?;
```

Comment:
Clippy is failing in a couple places:

<img width="832" alt="image" src="https://github.com/eq-lab/Slender/assets/12034486/ee1edbf0-c843-48e7-81eb-108db2e71165">

Reply:
I think we're ready to start github actions on PR and merge into master & develop